### PR TITLE
Form branch

### PR DESCRIPTION
**Pull request by Bhekisile**
Add form to html file and style it
Add form validation

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -401,3 +402,120 @@ h4 {
...
+  background-position: bottom right;
+}
+
+input:invalid {
+  border: 2px dashed red;
+}
+
+input:valid {
+  border: 2px solid #36b37f;
+}
```
Comment (braincee) [inline style.css]: - [ ] Good job implementing the input fields with placeholders. 👍 
But there's is an issue I highly recommend you to change. I found this dashed red around the placeholder text which is not part of the required figma design. I kindly suggest removing the dashed red.

_**REQUIRED FIGMA DESIGN**_

Screenshot (113)


_**YOUR DESIGN**_

Screenshot (114)

File: style.css [hunk 2]
```diff
@@ -401,3 +402,120 @@ h4 {
 .btn:focus {
   background-color: #008552;
 }
```
Comment (braincee) [inline style.css]: - [ ]  Great job using the correct button type and text. 👍 I think there is a little improvement you need to make to the UI. The Figma design provides the colours for different states of the button `hover`, `enabled` and `pressed` and according to the project requirements, you are supposed to stick to these designs as much as you can. I found out that the `.btn:enabled` and `.btn:focus` classes has overridden the functionality of the `btn:hover` class. I highly recommend removing `.btn:enabled` and `.btn:focus` inorder for the `hover` effect functionality to work. Which means there is no need adding `.btn:enabled` and `.btn:focus` classes. 😄

Screenshot (116)

File: index.html [hunk 3]
```diff
@@ -8,206 +9,244 @@
...
-                            <li class="css">CSS</li>
-                            <li class="js">JavaScript</li>
-                            <li class="html">HTML</li>
-                        </ul>
-                    </div>
-                    <button type="button" class="btn">See project</button>
-                </section>
+
+        <section class="works">
+            <header>
```
Comment (braincee) [inline index.html]: - [ ] [OPTIONAL] I found out that all the `section` tags are wrapped around the `header` tag which is a wrong implementation of the HTML semantic tag. The `header` tag should wrap around only the `nav` tag. This is a [link](https://developer.mozilla.org/en-US/docs/Glossary/Semantics) for HTML semantic tags. 😄

**Review by braincee — CHANGES_REQUESTED** [review]
Hello ✋, I am @braincee your Code Reviewer

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

so-close

HIGHLIGHTS
------

 - GitHub flow rightfully used ✅
 - Meaningful title and summary for PR ✅
 - Good implementation of pair-programming ✅
 - No linter errors ✅

REQUIRED CHANGES ♻️
------
**_Please check the comments under the review._**

OPTIONAL CHANGES
--------
**_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._**

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @braincee in your question so I can receive the notification.

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._

----

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -391,13 +392,118 @@ h4 {
...
 
 .btn:hover {
-  background-color: #36b37f;
+  background-color: #008552;
 }
```
Comment (AbelG101) [inline style.css]: - Great job 👏 making the changes requested by the previous reviewer but make sure you're following the Figma design as I noticed you only implemented the hover effect and that should be `#36B37F` and not `#008552` ❌. Kindly make sure you've also implemented the `:active` (when clicked) effect as well which should be  `#008552`.

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @Bhekisile 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a great summary ✔

 ## Required Changes ♻️
- _Check the comments under the review._

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by joel-kalema — APPROVED** [review]
Hi @Bhekisile 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
Linters are passing 👍 
The project is running successfully 👍 
PR is well descriptive 👍 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@joel-kalema** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**